server/__main__.py

```diff
@@ -55,22 +56,16 @@ def _retrieve_keycloak_public_key(keycloak_url: str, realm: str) -> str:
         response = requests.get(keycloak_url + "/realms/" + realm)
         response.raise_for_status()
     except:
-        logging.getLogger("werkzeug").warning("Failed to retrieve public key from Keycloak server.")
+        logger.warning("Failed to retrieve public key from Keycloak server.")
         return ""
-    logging.getLogger("werkzeug").info("Retrieved public key from Keycloak server.")
+    logger.info("Retrieved public key from Keycloak server.")
     return response.json()["public_key"]
 
 
-def _set_up_log_format() -> None:
-    """Set up the logging format."""
-    FORMAT = '%(asctime)s -- %(message)s'
-    logging.basicConfig(format=FORMAT)
-
-
 if __name__ == '__main__':
     vals = script_args.request_and_get_script_arguments("Run the Fleet Protocol v2 HTTP API server.")
     config = vals.config
-    _set_up_log_format()
+    configure_logging("./config/logging.json")
```

Comment:
Move to constant. Are paths in python relative to script path or working directory?

Reply:
To the CWD.